extendr-api/src/robj/tests.rs

```diff
@@ -92,21 +94,87 @@ fn test_from_robj() {
     }
 }
 
-#[test]
-fn test_try_from_robj() {
+#[rstest]
```

Comment:
Can you put the tests inside a test module locked behind a config flag?

Reply:
These tests should already be behind a test-flag,
```rust

#[cfg(test)]
mod tests;
```